modules/reference/pages/systems/system-details.adoc

```diff
@@ -70,6 +70,9 @@ The IP address of the client.
 IPv6 Address:::
 The IPv6 address of the client.
 
+Minion Id:::
+If the client is a salt minion, the minion id value is shown.
```

Comment:
```suggestion
On salt clients only, shows the minion identification value.
```

Reply:
That's the difference outcome when *you* are a mother language and a great writer......but I don't!! :wink:

Reply:
I still love your work, Dario <3